kvpress/presses/composed_press.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from kvpress.presses.base_press import BasePress
+
+
+@dataclass
+class ComposedPress(BasePress):
+    """
+    Chain multiple presses together to create a composed press
+    """
+
+    presses: list[BasePress]
+
+    def forward_hook(self, module, input, kwargs, output):
+        self.compression_ratio = 1
```

Comment:
I'd initialize `self.compression_ratio` parameter in __post_init__

Reply:
why ?